ui/language_utils.js

```diff
@@ -55,12 +59,27 @@ shaka.ui.LanguageUtils = class {
       }
     };
 
-    const getCombination = (language, rolesString, label) => {
+    const getCombination = (language, rolesString, label, channelsCount) => {
+      const keys = [
+        language,
+        rolesString,
+      ];
+      if (showAudioChannelCountVariants && channelsCount != null) {
+        keys.push(channelsCount);
```

Comment:
If I understand correctly, this means that now each separate channelsCount int will have a different button on the selector. However, only surround sound will show up in the label.
So (for example), if there is 1-channel english and 2-channel english, they will show up as "English" and "English", correct? Instead of "English Mono" and "English Stereo".

Reply:
Yes, that is correct, but I have not seen any stream that mixes mono and stereo for the same language, however I have seen mono/stereo with 5.1. I have not added the mono and stereo labels because right now no player shows them, but the surround ones do.

Reply:
Okay. I guess there's no point in putting in the effort to account for that unless someone complains about it.

Reply:
I have the same opinion